var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sensei(CMakePackage):
+    """SENSEI is a platform for scalable in-situ analysis and visualization.
+    Its design motto is 'Write once, run everywhere', this means that once
+    the application is instrumented with SENSEI it can use existing and
+    future analysis backends. Existing backends include: Paraview/Catalyst,
+    Visit/Libsim, ADIOS, Python scripts, and so on."""
+
+    homepage = "https://sensei-insitu.org"
+    url      = "https://gitlab.kitware.com/sensei/sensei/-/archive/v3.1.0/sensei-v3.1.0.tar.gz"
+    git      = "https://gitlab.kitware.com/sensei/sensei.git"
+
```

Comment:
@sshudler Do you want to add yourself and colleagues here as package maintainers? This will help us to ping the right people when community PRs provide additions to the SENSEI spack package.
```suggestion
maintainers = ['sshudler']
```